client/src/components/App.jsx

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import { CssBaseline } from '@mui/material/';
+import ThemeExample from './ThemeExample.jsx';
+import TopBar from './TopBar.jsx';
+import SongCardContainer from './SongCardContainer.jsx';
+import NavBar from './NavBar.jsx';
+import theme from '../themes/default.jsx';
 
-export default function() {
+
+export default function App() {
   return (
-    <div>
-      Hello World!
-    </div>
+    <ThemeProvider theme={theme}>
+        <CssBaseline/>
+        {/* <ThemeExample /> */}
+        <TopBar />
+        <SongCardContainer />
```

Comment:
:I assume this gets replaced with <SongCardList /> and was just used as a placeholder to test the <SongCard /> ?